hasher-matcher-actioner/terraform/fetcher/main.tf

```diff
@@ -205,7 +205,7 @@ resource "aws_iam_role_policy_attachment" "fetcher_trigger" {
 ### Config storage ###
 
 resource "aws_dynamodb_table" "threatexchange_config" {
-  name         = "${var.prefix}ThreatExchangeConfig"
```

Comment:
This is tested and works? I had though json keys cant include `-`

Reply:
? Not heard that, "-" appears to be valid: https://stackoverflow.com/questions/8676011/which-characters-are-valid-invalid-in-a-json-key-name

. I did indeed test it end to end, and records are in by db
![image](https://user-images.githubusercontent.com/1654004/114438145-3b6a3c00-9b7c-11eb-86b8-7109bfc0454e.png)